bool get_package_type
---
string pack_type